tests/checkpoint/test_checkpoint_result_format.py

```diff
@@ -359,9 +359,8 @@ def test_sql_result_format_in_checkpoint_pk_defined_one_expectation_complete_out
         dict_to_update_checkpoint=dict_to_update_checkpoint,
     )
 
-    result: CheckpointResult = context.run_checkpoint(
-        checkpoint_name="my_checkpoint",
-    )
+    checkpoint = context.get_checkpoint("my_checkpoint")
```

Comment:
I'll do a simple find and replace to make this `context.checkpoints.get` once the factory is set up